util/fetch/Cargo.toml

```diff
@@ -8,11 +8,11 @@ authors = ["Parity Technologies <admin@parity.io>"]
 
 [dependencies]
 futures = "0.1"
-hyper = "0.11"
-hyper-rustls = "0.11"
+hyper = "~0.12.9"
```

Comment:
Do we really need to be this specific?

Reply:
I can't remember exactly what but I'm depending on something that was added in `0.12.9`.

Reply:
Seems to work with `hyper v0.12.11` (which is what I got after changing to `"0.12"` in Cargo.toml)